Makefile

```
@@ -18,7 +18,9 @@ HEAVY_BADGER_TOOL= heavy-badger
 
 ALL_PACKAGES = ./...
 MOCKS_PACKAGE = github.com/insolar/insolar/testutils
-GOBUILD ?= go build
+GOBUILD ?= GO111MODULE=on go build -mod=vendor
```

```suggestion
export GOFLAGS='-mod=vendor'
export GO111MODULE=on
export CGO_ENABLED=1
```
and i suggest to use explicit `go build`/`go test` in code.